### PR TITLE
OLMIS-7655: Updated Input With Suggestions

### PR DESCRIPTION
**Pull request by dariapionk**
Updated Input after QA
OLMIS-7655

[OLMIS-7655]: https://openlmis.atlassian.net/browse/OLMIS-7655?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/react-components/inputs/input-with-suggestions.jsx [hunk 1]
```diff
@@ -14,46 +14,57 @@
...
 
 import React, { useState, useEffect } from 'react';
-import Input from './input';
+import Input from '../../react-components/inputs/input';
 
 
-const InputWithSuggestions = ({ data, onClick, sortFunction, displayValue, ...props }) => {
+const InputWithSuggestions = ({ data, onClick, displayValue, placeholder, ...props }) => {
+
+    const filterValues = (values, filterValue) => {
```
Comment (sniedzielski) [inline src/react-components/inputs/input-with-suggestions.jsx:22]: Can we shorten this line with return statement and rewrite this part of code as: 
 ```
    const filterValues = (values, filterValue) => {
        return filterValue !== '' ? values.filter((element) => { 
            return element.name.toLowerCase().indexOf(filterValue.toLowerCase()) > -1; 
        }) : values;
    };
```

File: src/react-components/inputs/input-with-suggestions.jsx [hunk 2]
```diff
@@ -63,15 +74,25 @@ const InputWithSuggestions = ({ data, onClick, sortFunction, displayValue, ...pr
...
-    <div>
-        { touched && resultsToShow }
-    </div>
+        <div style={{display: 'flex', gap: '10px', 'alignItems': 'center'}}>
+            <Input
+                {...props}
+                onChange={onChange}
+                value={inputValue}
+                onFocus={() => {setTouched(true)}}
+                placeholder={placeholder ? placeholder : 'Select an Option'}
```
Comment (sniedzielski) [inline src/react-components/inputs/input-with-suggestions.jsx]: placeholder={placeholder ? placeholder : 'Select an Option'} -> placeholder={placeholder || 'Select an Option'}
I think this shorten expression should work. I tested on browser's snippets with 'const/var placeholder' as '', null, particular value and undefined. For null, '', undefinied should be 'Select an option' . And for particular value we have particular value
Please check this if you have time.

**Review by sniedzielski — COMMENTED** [review]
I left two comments @dariapionk . Please refer to them.

**Comment by dariapionk** [comment]
@sniedzielski I corrected the code, can you check If it looks good now?

**Review by sniedzielski — APPROVED** [review]
@dariapionk LGTM.

**Pull request merged**